### PR TITLE
Adds few trivial things to make build for lnbits easier

### PR DESCRIPTION
**Pull request by arcbtc**
Related to this https://github.com/lnbits/cashu/pull/16

### CODE REVIEW

**Comment by arcbtc** [comment]
I also seem to have updated quasar and qrcode along the way

File: src/pages/WalletPage.vue [hunk 1]
```diff
@@ -2734,7 +2734,7 @@ export default {
     }
 
     // Clear all parameters from URL without refreshing the page
-    window.history.pushState({}, document.title, "/");
+    window.history.pushState({}, document.title, window.location.href);
```
Comment (callebtc) [inline src/pages/WalletPage.vue:2737]: This doesn't work

File: quasar.config.js [hunk 2]
```diff
@@ -44,10 +44,9 @@ module.exports = configure(function (ctx) {
 
     // Full list of options: https://v2.quasar.dev/quasar-cli-webpack/quasar-config-js#Property%3A-build
     build: {
-      vueRouterMode: "history", // available values: 'hash', 'history'
-
+      vueRouterMode: "hash", // available values: 'hash', 'history'
```
Comment (callebtc) [inline quasar.config.js:47]: Why is this `hash` necessary? It produces a `/#/` at the end of the url.

**Comment by callebtc** [comment]
I don't understand how this PR changes anything except for adding the `hash` history mode (which I'm not sure what it does.

**Pull request closed without merging**